webapp/src/shared/components/Header/Header.tsx

```diff
@@ -62,8 +87,43 @@ export default function Header({
       {children}
     </Text>
   );
+  let style: React.CSSProperties | undefined;
+  let statusElement: JSX.Element | undefined = undefined;
+  if (statusVariant) {
+    if (statusVariant !== 'button') {
+      statusElement = (
+        <div className="header-status">
+          {statusVariant && <Status variant={statusVariant} />}
+        </div>
+      );
+    } else {
+      if (buttonProps !== undefined) {
+        if (!windowIsBig) {
+          style = {
```

Comment:
This whole thiong is here (and inmainTabButtons as well) to not have the composition look like a d**k such as this
![not_two_balls](https://user-images.githubusercontent.com/2774420/201544422-73e708bd-0098-4071-99ed-56e7f397a7ea.png)

Reply:
I wouldn't worry too much merging it as is in this PR. We may have to explore other ideas but for this PR I think is good enough

Reply:
I've been thinking about this, what do you think about moving this exit button inside the edit page? This way we would avoid:
1. Having to inline styles and calculate translations etc
2. Simplifying mobile UI since one button is leaner and we gain more space for other content

So, a normal user will see this exit button and, an owner, admin etc. would see the edit chatroom option and would find the option to exit inside that page.

Reply:
That page already has two buttons, and they are stacked. You refused to accept smaller buttons. How would you make the design of the page with three buttons look like?

Reply:
I removed the two buttons, but I'm not convinced it's a good idea.
Maybe the link in the header title text brings to the edit chat page, and there is just one red button and it is, like for normal users, just to leave the chat?

Reply:
The latest proposal is an implementation of my last idea from the message above.

Reply:
Agreed with [this](https://github.com/42AGV/ft_transcendence/pull/262#discussion_r1022814381) and done